db/migrate/20210720221817_add_draft_count_to_user_stat.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDraftCountToUserStat < ActiveRecord::Migration[6.1]
+  def change
+    add_column :user_stats, :draft_count, :integer, default: 0, null: false
```

Comment:
We need a data migration here. Otherwise, a draft count of 0 shows up for everyone until a draft has been created/removed.

Reply:
I added a post migration.